blivet/devices/md.py

```diff
@@ -202,7 +202,7 @@ def level(self, value):
     def sector_size(self):
         if not self.exists:
             # Least common multiple of parents' sector sizes
-            return reduce(lambda a, b: a * b // math.gcd(a, b), [p.sector_size for p in self.parents])
+            return reduce(lambda a, b: a * b // math.gcd(a, b), [int(p.sector_size) for p in self.parents])
```

Comment:
This should go to the previous commit, but no big deal, I think.

Reply:
Fixed.